Ktisis/Configuration.cs

```diff
@@ -21,6 +23,26 @@ public class Configuration : IPluginConfiguration {
 
 		public bool DrawLinesOnSkeleton { get; set; } = true;
 		public float SkeletonLineThickness { get; set; } = 2.0F;
+        public UInt32 CategoryColor(Bone bone)
```

Comment:
Indent slip here.

Reply:
Ah yes, please don't review it just now, I'm going to review all my changes, making a clean clone and clean the extra folders I have. I think I restarted coding with a local that wasn't rebased, now i'm trying to clean this mess

Reply:
If you need help, ask. (Though preferably not in this discussion -- I'm `Fayti1703#5459` on Discord, if you use that.)

Reply:
I might take the offer ^^' i sent a request.
Actually it is not so bad, maybe I actually made those mistakes, but when i saw the double `using` (below), I thought i made another mistake.